tests/zenko_tests/node_tests/smoke_tests/vault_admin_and_IAM_API_tests/adminApi.js

```diff
@@ -13,16 +11,21 @@ const accountInfo = {
 let iamClient = null;
 
 describe('Accounts: ', () => {
-    it('should be able create, generate credentials and delete an account', done => async.series([
-        next => clientAdmin.createAccount(accountName, accountInfo, next),
-        next => clientAdmin.generateAccountAccessKey(accountName, (err, res) => {
-            if (err) {
-                return next(err);
-            }
-
-            iamClient = VaultClient.getIamClient(res.id, res.value);
-            return next(null);
-        }),
-        next => VaultClient.deleteVaultAccount(clientAdmin, iamClient, accountName, next),
-    ], done));
+    it('should be able create, generate credentials and delete an account', done => {
+        async.series([
+            next => clientAdmin.createAccount(accountName, accountInfo, next),
+            next => clientAdmin.generateAccountAccessKey(accountName, (err, res) => {
+                if (err) {
+                    return next(err);
+                }
+                iamClient = VaultClient.getIamClient(res.id, res.value);
+                return next(null);
+            }),
+            next => {
+                VaultClient.deleteVaultAccount(clientAdmin, iamClient, accountName)
```

Comment:
You can easily move to async/await here 
```suggestion
                async() => VaultClient.deleteVaultAccount(clientAdmin, iamClient, accountName)
```